src/app/zap-templates/zcl/operational-credentials-cluster.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>Operational Credentials</name>
+    <code>0xfc03</code> // Change to correct clusterID once it is decided in the spec #2396
+    <define>OPERATIONAL_CREDENTIALS_CLUSTER</define>
+    <description>This cluster is used to add or remove Operational Credentials on a Commissionee or Node, as well as manage the associated Fabrics.</description>
+
+    <attribute side="server" code="0x0001" define="FABRICS" type="ARRAY" entryType="FabricDescriptor" length="254" writable="false" optional="false">fabrics list</attribute>
+
+    <command source="client" code="0x09" name="UpdateFabricLabel" optional="false">
+      <description>This command SHALL be used by an Administrative Node to set the user-visible Label field for a given Fabric, as reflected by entries in the Fabrics attribute.</description>
+      <arg name="Label" type="OCTET_STRING" length="32"/> //TODO: change this to CHAR_STRING once it is supported #6112
+    </command>
+
+    <command source="client" code="0x0a" name="RemoveFabric" optional="false">
+      <description>This command is used by Administrative Nodes to remove a given Fabric and delete all associated
+fabric-scoped data.</description>
+      <arg name="FabricId" type="FABRIC_ID"/>
+      <arg name="NodeId" type="NODE_ID"/>
```

Comment:
Are you sure this is correct? If I'm looking at the right place in the spec `RemoveFabric` command only takes `FabricIndex`.

Reply:
I am aware of that. Please see my response to Michael above where I explain the reason behind this. Ok short, without addOptCert + fabric index implemented, this was the best way I could find for commissioner to uniquely identify which fabric they want removed. Once we implement the rest of this cluster, I will change the parameters of this command to take fabric index.